components/game/tetris.tsx

```diff
@@ -0,0 +1,124 @@
+import { useEffect, useState } from 'react';
+
+import * as TetrisConsts from 'constants/tetris';
+
+/**
+ * @brief: fieldToJsxElement: Convert a field of coords to a JSX element for
+ * render
+ * @param[in]: field - Field to render
```

Comment:
Nitpicking but probably want to add in the `grid` input as well. I think we can also remove the `[in]` for the params? I know that in C/C++ you need to specify either `[in/out]` because of pass-by-ref but in JS there's no such thing :thinking: